lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1854,13 +1856,16 @@ group_config(ecdsa = Type, Config) ->
     [{sign_verify, SignVerify}, {pub_priv_encrypt, PubPrivEnc} | Config];
 group_config(Type, Config) when Type == ed25519 ; Type == ed448 ->
     TestVectors = eddsa(Type),
-    [{sign_verify,TestVectors} | Config]; 
+    Generate = lists:map(fun({Curve, _Hash, Priv, Pub, _Msg, _Signature}) ->
+                             {eddsa, Curve, Priv, Pub}
+                         end, TestVectors),
+    [{sign_verify,TestVectors}, {generate, Generate} | Config]; 
 group_config(srp, Config) ->
     GenerateCompute = [srp3(), srp6(), srp6a(), srp6a_smaller_prime()],
     [{generate_compute, GenerateCompute} | Config];
 group_config(ecdh, Config) ->
     Compute = ecdh(),
-    Generate = ecc(),
+    Generate = ecc() ++ ecc(x25519) ++ ecc(x448),
```

Comment:
For x25519 and x448, this appending bypasses the check for curve support in ecc/0.  It causes this test case to fail on machines with older OpenSSL cryptolib that lacks x25519 and x448 in a form usable for OTP.
So please move the contents of ecc/1 into the list TestCases in ecc/0.

Reply:
Thanks for testing it. I have moved x448 & x25519 under ecc/0.